### PR TITLE
SML catchup (11-15)

### PR DESCRIPTION
**Pull request by hypirion**
Been too busy because 🎅🏼 , so I'm some days behind on Standard ML. More will be coming over the next days I presume.

### CODE REVIEW

File: days/day-13/solutions/day13.sml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    end
+
+fun crt v jump (lst as (a, n) :: xs) =
+    if v mod n = a then
+        crt v (jump * n) xs
+    else
+        crt (v + jump) jump lst
+  | crt v jump [] = v
+
+fun solve2 buses =
```
Comment (Arxcis) [inline days/day-13/solutions/day13.sml:21]: So short ! :+1:

File: days/day-15/solutions/day15.sml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+in
+
+fun foldli f init l =
+    #2 $ List.foldl (fn (v, (i, acc)) => (i+1, f (i, v, acc))) (0, init) l
+
+fun butlast lst = List.take (lst, List.length lst - 1)
+
+fun run init n =
+    let val withInit = foldli (fn (i, v, m) => IM.insert v i m) IM.empty $ butlast init
+        fun recur k prev m =
```
Comment (Arxcis) [inline days/day-15/solutions/day15.sml:14]: Recursion on this one??!! Superbadass!

**Review by Arxcis — APPROVED** [review]
Wow impressive work :smiley: Todays MVP by far :clap: Bringing us from 93 -> 98 solutions in total. Almost  :100: now :sweat_smile: Approved!

File: days/day-15/solutions/day15.sml (hunk 2 above)
Comment (hypirion) [reply days/day-15/solutions/day15.sml:14]: Hey, I don't have any choice :)

**Pull request merged**